code/game/machinery/cryopod.dm

```diff
@@ -290,6 +290,11 @@
 					continue
 				O.loc = src
 
+	for(var/obj/machinery/computer/cloning/cloner in machines)
+		for(var/datum/dna2/record/R in cloner.records)
+			if(occupant.mind == locate(R.mind))
```

Comment:
[Darn it cloning code, stop being weird](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/machinery/computer/cloning.dm#L424-L425)